lib/db-sync-progress.js

```diff
@@ -24,19 +26,26 @@ module.exports = function (db, opts) {
   return stream
 
   function onFeed (feed) {
-    feeds.push(feed)
+    if (feeds.has(feed)) return
```

Comment:
Is this working as-is? I'd expect this to be

```suggestion
    if (feeds.has(feed.key.toString('hex'))) return
```

I was going to say use `feed.key` as the map key, but I just checked and it turns out buffers as keys only work if you pass in the *same* buffer reference. Using the hex key is, I think, the safest.